sandbox/lua/lua_sandbox_interface.c

```diff
@@ -100,13 +104,18 @@ int timer_event(lua_sandbox* lsb, long long ns)
 
     lua_pushnumber(lua, ns);
     if (lua_pcall(lua, 1, 0, 0) != 0) {
+        const char* errmsg = lua_tostring(lua, -1);
         char err[LSB_ERROR_SIZE];
         size_t len = snprintf(err, LSB_ERROR_SIZE, "%s() %s", func_name,
-                              lua_tostring(lua, -1));
+                              errmsg);
         if (len >= LSB_ERROR_SIZE) {
           err[LSB_ERROR_SIZE - 1] = 0;
         }
-        lsb_terminate(lsb, err);
+        // Don't terminate if we're aborting so we preserve data on exit.
+        const char* tail = errmsg + strlen(errmsg) - 7; // Last 7 chars of err msg.
```

Comment:
Out of bounds if the error message is less than seven characters long (you can use lua_tolstring to grab the length when retrieving errmsg or just grab it off the end of err (which is always longer than 7)